cfgov/jinja2/v1/owning-a-home/_layouts/related-links.html

```diff
@@ -0,0 +1,180 @@
+                       Row 1: Know the Process, Tools and Resources, More Information
+                       Row 2: Need help?, About the CFPB
+                       Only the first two sections have links that change by page.
+
+                       Since most of the content is standard, we just pass in
+                       arrays containing the links for the first two sections on the page,
+                       in the below format:
+
+                       [
+                         {
+                           'desc': 'description for link'
+                           'urls': [
+                            {
+                              'link': 'link label',
+                              'url': 'link url'
+                            }
+                           ]
+                          }
+                        ]
+
+   ========================================================================== #}
+
+{% macro render(process_links, tools_links) %}
+
+{% endmacro %}
+
+
+{# ==========================================================================
+
+   render_row(data)
+
+   ==========================================================================
+
+   Description:
+
+   Render a row of related-link sections.
+
+   sections:          An array of section objects. Each section object
+                      contains a title and an array of link objects. It
+                      can also contain an optional col property, determining
+                      how many columns the section spans (defaults to 1-3);
+
+   ========================================================================== #}
+
+{% macro render_row(sections) %}
+    <div class="content-l">
+      {% for section in sections %}
+        {{render_section(section)}}
+      {% endfor %}
+    </div>
+{% endmacro %}
+
+
+{# ==========================================================================
+
+   render_section(data)
+
+   ==========================================================================
+
+   Description:
+
+   Render a section of related links.
+
+   data:              An object, containing:
+                      - title (req): title of section
+                      - links (req): An array of link objects. Each object should include a
+                                     description, a label, and a url.
+                      - type (opt): Link icon
+                      - col (opt): Column width for this section. Defaults to 1-3.
+
+
+   ========================================================================== #}
+
+{% macro render_section(data) %}
+  <div class="content-l_col content-l_col-{{data.col|default('1-3')}}">
+    <h2 class="header-slug">
+      <span class="header-slug_inner">{{data.title}}</span>
+    </h2>
+    <ul class="related-links_list">
+      {% for link in data.links %}
+        <li>
+          <p class="short-desc">
+            {{link.desc|safe}}
+          </p>
+          <ul class="list list__links m-list__unstyled">
```

Comment:
This files has been removed in https://github.com/cfpb/cfgov-refresh/pull/3165/commits/19a504479db4aac3d7787a541bf6f22615d9a181.